src/convert.rs

```diff
@@ -368,9 +370,12 @@ pub fn elf_to_tbf<W: Write>(
             && input.ehdr.entry < (section.shdr.addr + section.shdr.size)
             && (section.shdr.name.find("debug")).is_none()
         {
-            // panic in case we detect entry point in multiple sections.
             if entry_point_found {
-                panic!("Duplicate entry point in {} section", section.shdr.name);
+                if disabled {
+                    println!("Duplicate entry point in {} section", section.shdr.name);
+                } else {
+                    panic!("Duplicate entry point in {} section", section.shdr.name);
+                }
```

Comment:
I don't understand why having a disabled app changes this logic. A comment here would be helpful.